tools/upgrade-loader-or-syncer-to-dm.md

```diff
@@ -0,0 +1,95 @@
+
+## Upgrade Syncer to Data Migration
+
+Syncer is a tool used to import data incrementally. The task Syncer executes is corresponding to the synchronization task with `incremental` `task-mode` in DM. The `syncer` processing unit feature of dm-worker in DM is corresponding to the Syncer feature.
+
+To upgrade Syncer to DM, you only need to generate the corresponding task configuration file and then use dmctl to execute the task.
+
+### Configuration change
+
+Syncer uses the TOML file to define the process-related operation parameters and synchronization task parameters; DM uses the YAML file to define task configuration file parameters.
+
+Taking the configuration options in [Data Migration Task Configuration File](../tools/dm-task-configuration-file-intro.md) as examples, the corresponding relationship of task configuration options between Syncer and DM is as follows:
+
+| Configuration in Syncer | Corresponding configuration in DM |
+| ---------------------- | --------------------------------- |
+| `server-id` | Transferred to `dm-worker.toml` |
+| `flavor` | Transferred to `dm-worker.toml` |
+| `enable-gtid` | Transferred to `dm-worker.toml` |
+| `auto-fix-gtid` | Transferred to `dm-worker.toml` |
+| `meta` | `meta` of `mysql-instances`. `binlog-name`/`binlog-pos` of `meta` in Syncer corresponds to that of `mysql-instances`. | 
+| `persistent-dir` | Deprecated |
+| `worker-count` | `worker-count` of `syncer` | 
+| `batch` | `batch` of `syncer` | 
+| `max-retry` | `max-retry` of `syncer` |  
+| `do-db` | `do-dbs` of `black-white-list`. The filtering feature of databases has been refactored. For detailed configuration, see `do-dbs` in [Data Migration Task Configuration File](../tools/dm-task-configuration-file-intro.md). |
+| `do-table` | `do-tables` of `black-white-list`. The filtering feature of tables has been refactored. For detailed configuration, see `do-tables` in [Data Migration Task Configuration File](../tools/dm-task-configuration-file-intro.md). |
+| `ignore-db` | `ingore-dbs` of `black-white-list`. The filtering feature of databases has been refactored. For detailed configuration, see `ignore-dbs` in [Data Migration Task Configuration File](../tools/dm-task-configuration-file-intro.md). | 
+| `ignore-table` | `ignore-tables` of `black-white-list`. The filtering feature of tables has been refactored. For detailed configuration, see `ignore-tables` in [Data Migration Task Configuration File](../tools/dm-task-configuration-file-intro.md). |
+| `skip-ddls` | Deprecated. Use `filters`. |
+| `skip-sqls` | Deprecated. Use `filters`. |
+| `skip-events` | Deprecated. Use `filters`. |
+| `skip-dmls` | Deprecated. Use `filters`. |
+| `route-rules` | `route-rules`|
+| `pattern-schema` | `schema-pattern` |
+| `pattern-table` | `table-pattern` |
+| `from` | `config` of `mysql-instances`. Should be consistent with the upstream MySQL information during the DM-worker deployment. |
```

Comment:
```suggestion
| `from` | `config` of `mysql-instances`. Keep it consistent with the upstream MySQL information during the DM-worker deployment. |
```

Reply:
The word "should" is a little bit too weak. Using an imperative sentence is clearer.